core/src/plugins/filed/BareosFdPluginLibcloud.py

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/python
```

Comment:
As most of it is a rewrite, I think we can put our bareos copyright header and put "original author: Alexandre Bruyelles" here